fuse/dl/lightning/pl_funcs.py

```diff
@@ -157,7 +157,7 @@ def epoch_end_compute_and_log_losses(pl: pl.LightningModule, mode: str, batch_lo
             else:
                 losses.append(elem[key])
         loss = mean(losses)
-        pl.log(f"{mode}.losses.{key}", loss, on_epoch=True)
+        pl.log(f"{mode}/losses.{key}", loss, on_epoch=True)
```

Comment:
might break compatibility..

Reply:
Lets add flag for backward Compatability